utils/release_tracking.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import re
+import os
+import subprocess
+import sys
+
+
+def run_cargo_build(path):
+    print(f"cargo build, path: {path}")
+    command = f'cargo build'
+    subprocess.check_output(command.split(), cwd=path)
+
+
+def clone_repo(clone_dir, repo_name, branch):
+    git_repo = f"https://github.com/parallaxsecond/{repo_name}.git"
```

Comment:
`repo_branches` `parallaxsecond_deps` or `repo_paths` are defined in main, so I think better not to hardcode them as a file global but just defined in main and in case we want to scale things then put them as a global thing

Reply:
I will use your suggestion and pass the git_repo as an argument

Reply:
Done